tests/setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import find_packages, setup
+
+setup(name="kafkatest",
+      version="0.8.3-SNAPSHOT",
```

Comment:
Are we sure we want to change this now? We probably shouldn't even be publishing to PyPI except on releases when this would change anyway. In fact, shouldn't we just try to grab it out of build.gradle or wherever it normally is so we don't have version information duplicated in more locations than necessary?